### DISCUSSION TITLE
Low refresh rate on Ubuntu

### DISCUSSION THREAD

**Discussion by marktiro** [atom/atom#22729] [Q&A]
Hi, I noticed that even though the system is set to 144Hz refresh rate, the editor is running under 60Hz nonetheless.
I am using an Intel integrated GPU and Ubuntu 21.04.
Does anyone know how to configure the editor to be get 144Hz refresh rate?

To be clear, I noticed it while scrolling and tested it with developer-tools of Chrome using the FPS meter.

**Comment by gifhuppp** [comment]
![Uploading IMG_20201120_084132.jpg…]()
hai
what do you country

**Reply by marktiro** [reply]
?

**Reply by icecream17** [reply]
Just spam, ignore
